CONTRIBUTING.md

```diff
@@ -123,6 +123,7 @@ Sonatype ecosystem.
 
 After a successful release, do not forget to:
 * add the changelog to the [releases page](https://github.com/apollographql/apollo-android/releases).
+* also paste the changelog to the `CHANGELOG.md` file
```

Comment:
Now that we have a nice CHANGELOG.md file, we can use that as source of truth and add there first?

Reply:
Totally! 👍